portfolio/src/main/webapp/second.html

```diff
@@ -0,0 +1,25 @@
+<html>
+
+<head>
+    <meta charset="UTF-8">
+    <title>About me</title>
+    <link rel="stylesheet" href="style.css">
+    <div class="topnav">
+        <a class="active" href="index.html">Home</a>
+        <a href="#second">About me</a>
+        <a href="projects.html">Projects</a>
+        <a href="skills.html">Skills</a>
+        <a href="contact.html">Contact</a>
+    </div>
+</head>
+<header id="second">
+    <h1> About me </h1><br>
+    <section id="seaction1">
+        <div class="box2" <p> My name is Siham Abdi and I am a student at the University of Minnesota studying computer science and Sociology. I love learning and trying new things. During my free time I watch shows, read, or skateboard. I grew up in a city 1.5 hours from
```

Comment:
Similarly here, once you have a lot of classes, you won't remember what "Box2" represents. Please make this class more descriptive.

Reply:
The "div" needs a closing brace. So 
```suggestion
        <div class="box2">
```

Reply:
We usually like to have elements on their own lines.
```suggestion
        <div class="box2"> 
          <p> My name is Siham Abdi and I am a student at the University of Minnesota studying computer science and Sociology. I love learning and trying new things. During my free time I watch shows, read, or skateboard. I grew up in a city 1.5 hours from
```